spyder/plugins/completion/kite/plugin.py

```diff
@@ -222,3 +224,16 @@ def _show_onboarding_file(self, onboarding_file):
             return
         self.set_option('show_onboarding', False)
         self.main.open_file(onboarding_file)
+
+    @Slot(str, object)
+    def _wrong_response_error(self, method, resp):
+        QMessageBox.critical(
+            self.main, _('Kite error'),
+            _("The Kite completion engine returned an unexpected result "
+              "for the request <tt>{0}</tt>: <br><br><tt>{1}</tt><br><br>"
+              "Please make sure that your Kite installation is correct. "
+              "In the meantime, Spyder will disable the Kite client to "
+              "prevent further errors. For more information, please "
+              "visit the <a href='https://help.kite.com/'>Kite help "
+              "center</a>").format(method, response))
```

Comment:
@andfoy I think instead of `response` it should be `resp` (or maybe we should change the param to be `response`?)